### PR TITLE
creating readme for bert-base-mongolian-uncased

### PR DESCRIPTION
**Pull request by bayartsogt-ya**
I am adding the model card for bert-base mongolian uncased.

Can you review this for me please!

### CODE REVIEW

**Comment by bayartsogt-ya** [comment]
@julien-c 
hi, I am wondering if you guys accepting new model cards?

File: model_cards/bayartsogt/bert-base-mongolian-uncased/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+language: "mn"
+tags:
+- bert
```
Comment (julien-c) [inline model_cards/bayartsogt/bert-base-mongolian-uncased/README.md:4]:
```suggestion
```
should be automatic

**Review by julien-c — APPROVED** [review]
Yes! Thanks for sharing

**Comment by julien-c** [comment]
If you'd like, it'd be awesome if you could add default input texts in Mongolian for https://github.com/huggingface/widgets-server/blob/master/DefaultWidget.ts (you can open a PR)

so the inference widget on your model pages is correctly populated

**Pull request merged**